### ISSUE TITLE
Changes Requested

### ISSUE THREAD

**Issue by ghost** [Mnv17/Todo-App#1]
Great job so far 🎉 

Can you please work on the following changes?

1. Deploy the application on vercel.
2. Complete the edit and delete functionality.
3. Add functionality to mark the todo as completed.
4. Once the user signs in the user should be able to see only their todos, and not the todos created by someone else.
5. Make UI prettier. Use any existing available material UI template and use icons for edit & delete.

**Comment by Mnv17** [comment]
Sure i'll do that. Can you please tell me the deadline for these changes ?
So i can work accordingly.

**Comment by ghost** [comment]
> Sure i'll do that. Can you please tell me the deadline for these changes ? So i can work accordingly.

What's your tentative timeline?
The goal is to do as fast as you can.
You can keep on improving your solution, but we want to reach to the first working solution as fast as possible, and improve it.

**Comment by ghost** [comment]
Also I would urge you to write better commits.
Use this tool: https://github.com/Everduin94/better-commits
Reference: https://github.blog/2022-06-30-write-better-commits-build-better-projects/

**Comment by ghost** [comment]
Hey @Mnv17 
Do you have any estimate timeline for the given amount of work?

**Comment by Mnv17** [comment]
@jinayshah86 Yes. I have been told that the deadline was 31st march. And i have updated the deployed link as well, actually i have added the link in readme file but now i have added in About section as well. Please check it once.
